src/overlays/gamestates/ovl_file_choose/z_file_choose.c

```diff
@@ -907,34 +915,49 @@ static void* sTitleLabels[3][9] = {
     { gFileSelPleaseSelectAFileFRATex, gFileSelOpenThisFileFRATex, gFileSelCopyWhichFileFRATex,
       gFileSelCopyToWhichFileFRATex, gFileSelAreYouSureFRATex, gFileSelFileCopiedFRATex, gFileSelEraseWhichFileFRATex,
       gFileSelAreYouSure2FRATex, gFileSelFileErasedFRATex }
+#endif
 };
 
-static void* sWarningLabels[3][5] = {
+static void* sWarningLabels[][5] = {
+#if OOT_NTSC
+    { gFileSelNoFileToCopyJPNTex, gFileSelNoFileToEraseJPNTex, gFileSelNoEmptyFileJPNTex, gFileSelFileEmptyJPNTex,
+      gFileSelFileInUseJPNTex },
+    { gFileSelNoFileToCopyENGTex, gFileSelNoFileToEraseENGTex, gFileSelNoEmptyFileENGTex, gFileSelFileEmptyENGTex,
+      gFileSelFileInUseENGTex },
+#else
     { gFileSelNoFileToCopyENGTex, gFileSelNoFileToEraseENGTex, gFileSelNoEmptyFileENGTex, gFileSelFileEmptyENGTex,
       gFileSelFileInUseENGTex },
     { gFileSelNoFileToCopyGERTex, gFileSelNoFileToEraseGERTex, gFileSelNoEmptyFileGERTex, gFileSelFileEmptyGERTex,
       gFileSelFileInUseGERTex },
     { gFileSelNoFileToCopyFRATex, gFileSelNoFileToEraseFRATex, gFileSelNoEmptyFileFRATex, gFileSelFileEmptyFRATex,
       gFileSelFileInUseFRATex },
+#endif
```

Comment:
an alternative, much shorter but obfuscated:
```c
#define WARNING_LABELS(language)                                                                                       \
    {                                                                                                                  \
        gFileSelNoFileToCopy##language##Tex, gFileSelNoFileToErase##language##Tex, gFileSelNoEmptyFile##language##Tex, \
            gFileSelFileEmpty##language##Tex, gFileSelFileInUse##language##Tex                                         \
    }
static void* sWarningLabels[][5] = {
    LANGUAGE_ARRAY(WARNING_LABELS(JPN), WARNING_LABELS(ENG), WARNING_LABELS(GER), WARNING_LABELS(FRA)),
};
```

I think I prefer how the PR is currently tbh